tests/test_vault.yml

```diff
@@ -61,6 +64,18 @@
               ip_sans: "127.0.0.1"
               alt_names: "example.com"
               exclude_cn_from_sans: true
+    - name: Configure PKI
+      import_role:
+        name: vault_pki
+      vars:
+        vault_token: "{{ vault_keys.root_token }}"
+        vault_pki_root_ca_name: "OS-TLS-ROOT"
+        vault_pki_intermediate_ca_name: "OS-TLS-INT"
+        vault_pki_root_create: False
+        vault_pki_intermediate_create: False
+        vault_pki_generate_certificates: True
+        vault_pki_write_pem_bundle: True
+        vault_pki_certificate_subject:
```

Comment:
It's probably time to verify that this has actually done what we wanted it to. Bare minimum would be checking for existence of the generated certs, ideally checking the trust chain too.

Reply:
TODO